This change allows ntp-wait to work with more versions
of NTP, since the format changed slightly. It can be removed
at the next upgrade since there will be no possibility 
of running an older ntp with the old format.

--- scripts/ntp-wait.in
+++ scripts/ntp-wait.in
@@ -20,8 +20,8 @@
     while(<Q>) {
       chomp;
       # the first line should be similar to:
-      # associd=0 status=0645 leap_none, sync_ntp, ...
-      if (/^asso?c?id=0 status=(\S{4}) (\S+), (\S+),/i) {
+      # status=0645 leap_none, sync_ntp, ...
+      if (/status=(\S{4}) (\S+), (\S+),/i) {
 	my $status = $1;
 	my $leap = $2;
 	my $sync = $3;
